### DISCUSSION TITLE
Is there documentation that list what each of the features do and what the parameters are?

### DISCUSSION THREAD

**Discussion by Chrispie** [diffplug/spotless#2075] [Q&A]
I looked at the readme and there is a [current-feature-matrix](https://github.com/diffplug/spotless?tab=readme-ov-file#current-feature-matrix).
But what I am looking for is a 1 or 2 liner of what each feature does with explanation of what the parameters do.
For now I have to click on each link and try to see from the code what it does but it is not efficient. 
Am I just missing this info or does it not exists? :)

**Comment by nedtwigg** [comment]
You probably want one of these

- https://github.com/diffplug/spotless/tree/main/plugin-gradle
- https://github.com/diffplug/spotless/tree/main/plugin-maven

Spotless just calls out to the formatters you pick, you'll have to look at their docs on their site. We link to their docs from the docs above to make that easy.
